source/models/row.py

```diff
@@ -1,15 +1,19 @@
 import os, json, uuid
-import time
 from outputs.exceptions import *
 from models.index import Index
 
 class Row:
-    def __init__(self, table, value = None):
+    def __init__(self, table, value = {}):
         self.table = table
         self.value = value
-        if self.table.table_metadata.primary_key not in value : 
-            value[self.table.table_metadata.primary_key] = str(uuid.uuid4().node)
-        self.primary_key = value[self.table.table_metadata.primary_key]
+        self.primary_key = self.__get_primary_key()
```

Comment:
You don’t need primary_key instance variable. Get_primary_key() method is sufficient

Reply:
ok i will remove it

Reply:
done